### PR TITLE
add descriptor in getinfo, closes #174

### LINKED ISSUE #174: Add the descriptors in getinfo
We keep the `deposit_descriptor`, `unvault_descriptor` and `cpfp_descriptor` around after parsing the configuration. It would be nice if the `getinfo` command returned them. Maybe like so:
```json
"descriptors": {
    "deposit": "",
    "unvault": "",
    "cpfp": ""
}
```

### PR DESCRIPTION
**Pull request by vaibhavgarg237**
Fixes #174

### CODE REVIEW

File: doc/API.md [hunk 1]
```diff
@@ -35,14 +35,15 @@ Display general information about the current daemon state.
...
-| `managers_threshold` | integer | Number of managers needed for spending the `unvault_tx`         |
+| Field                | Type         | Description                                                            |
+| -------------------- | ------------ | ---------------------------------------------------------------------- |
+| `blockheight`        | integer      | Current block height                                                   |
+| `network`            | string       | Answer can be `mainnet`, `testnet`, `regtest`                          |
+| `sync`               | float        | The synchronization progress as percentage (`0 < sync < 1`)            |
+| `version`            | string       | Version following the [SimVer](http://www.simver.org/) format          |
+| `vaults`             | integer      | Current number of vaults (unconfirmed are included)                    |
+| `managers_threshold` | integer      | Number of managers needed for spending the `unvault_tx`                |
+| `descriptors`        | string array | `deposit_descriptor`, `unvault_descriptor` & `cpfp_descriptor`         |
```
Comment (darosior) [inline doc/API.md:46]: It's not a string array, it's an object (`{}`, not `[]`).

How about something like:
```
 | descriptors        | object       |  Three `string` entries: `deposit`, `unvault` and `cpfp` for the three Miniscript descriptors |
 ```

**Review by darosior — COMMENTED** [review]
Great, the code looks good to me! Just a small nit on the documentation.

File: doc/API.md (hunk 1 above)
Comment (vaibhavgarg237) [reply doc/API.md]: Pushed the required changes

**Review by darosior — COMMENTED** [review]
ACK the code. However this needs a test.

Can you extend the `test_getinfo` functional test to assert that the descriptors returned by the `getinfo` RPC call are the same as the ones we configured the `revaultd`s with.
Hint: look at were we call `mscompiler` to get those descriptors and if you can make them attributes of `RevaultD` instances.

**Comment by vaibhavgarg237** [comment]
> test

Made the required changes in test_misc.py, extending the descriptors

**Comment by darosior** [comment]
utACK 3a6896f6892816754c25529331047ebf5fbc8312

**Pull request merged**